.travis.yml

```diff
@@ -1,3 +1,29 @@
 language: node_js
+
 node_js:
   - "6"
+
+services:
+  - docker
+
+env:
+  GOPATH: ${TRAVIS_BUILD_DIR}/_workspace
+
+hosts:
+    - alice.cozy.io
+    - bob.cozy.io
+    - charlie.cozy.io
```

Comment:
The hosts are not actually used for now, but I figured we will need them later to tests CORS and auth (with karma) and didn't want to forget how it is done in travis.